src/js/components/DataTable/Header.js

```diff
@@ -500,8 +504,13 @@ const Header = forwardRef(
                       ? `${widths[property]}px`
                       : undefined,
                     boxSizing: onResize ? 'border-box' : undefined,
-                    position: onResize ? 'relative' : undefined,
+                    // Don't override positioning when DataTable has pin prop
+                    position:
+                      onResize && !columnPin && !pinProp
+                        ? 'relative'
+                        : undefined,
                     overflow: onResize ? 'visible' : undefined,
+                    zIndex: onResize && columnPin ? PINNED_ZINDEX : undefined,
```

Comment:
The `zIndex` in `StyledDataTableCell` is set to `z-index: ${Object.keys(props.pin).length};`

How is this interacting with that? It seems like it would override it which makes me wonder about multiple pins

Reply:
ah good point I could see this causing conflict